server/src/mindustry/server/ServerControl.java

```diff
@@ -666,7 +666,7 @@ private void registerCommands(){
             if(info != null){
                 info.lastKicked = 0;
                 info("Pardoned player: {0}", info.lastName);
-            } else{
+            }else{
                 err("That ID hasn't been votekicked!");
```

Comment:
Might be more accurate to say "That ID can't be found" instead.

Reply:
oke